NuRadioReco/framework/event.py

```diff
@@ -137,6 +138,44 @@ def set_id(self, evt_id):
     def get_run_number(self):
         return self.__run_number
 
+    def get_waveforms(self, station_id=None, channel_id=None):
+        """
+        Returns the waveforms stored within the event.
+
+        You can specify the station and channel id to get specific waveforms.
+        If you do not specify anything you will get all waveforms.
+
+        Parameters
+        ----------
+        station_id: int (Default: None)
+            The station id of the station for which the waveforms should be returned.
+            If `None`, the waveforms of all stations are returned.
+        channel_id: int or list of ints (Default: None)
+            The channel id(s) of the channel(s) for which the waveforms should be returned.
+            If `None`, the waveforms of all channels are returned.
+
+        Returns
+        -------
+        times: np.ndarray
+            A numpy array containing the times of the waveforms.
+        waveforms: np.ndarray
+            A numpy array containing the waveforms.
```

Comment:
Can you add here what it actually returns? I assume it is nChannel x amplitudes ?

Reply:
you mean the shape?

Reply:
Like this or is this now to confusing?